README.md

```diff
@@ -9,12 +9,6 @@ Orange3-Timeseries
 Orange add-on for analyzing, visualizing, manipulating, and forecasting time
 series data.
 
-**License: CC-BY-NC-3.0**
-
-In order to use this package commercially, please obtain a [Highcharts] license.
-
-[Highcharts]: http://www.highcharts.com/
-
```

Comment:
I removed this paragraph instead of replacing it; I checked other add-ons and they do not mention license at all. This was here, presumably, to warn commercial users about the highcharts license.